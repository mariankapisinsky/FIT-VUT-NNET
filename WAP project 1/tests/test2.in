5,7,2131345646,9,4,13,12415486