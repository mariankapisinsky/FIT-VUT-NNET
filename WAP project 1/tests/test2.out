preorder
5,4,7,2131345646,9,13,12415486
inorder
4,5,7,9,13,12415486,2131345646
postorder
4,12415486,13,9,2131345646,7,5